edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Docker.Test/EdgeAgentDockerModuleTest.cs

```diff
@@ -121,7 +121,7 @@ public void MixedIModuleImplEqualsTest()
 
             var fullRuntimeModule = CreateEdgeAgentDockerRuntimeModule(new DockerConfig(
                 "Foo",
-                createOptions: "{\"ignore\": \"me\"}",
```

Comment:
revert it.

Reply:
Oh.  master already have this change.  I don't understand why this change shows up?